internal/facets/contactentity/contactentity.go

```
@@ -0,0 +1,451 @@
+		}
+	}
+
+	return contacts
+}
+
+func IsContactEntityType(value string) bool {
+	switch strings.TrimSpace(value) {
+	case "http://xmlns.com/foaf/0.1/Person",
+		"http://xmlns.com/foaf/0.1/Organization",
+		"https://schema.org/Person",
+		"https://schema.org/Organization",
```

golangci-lint (2.12.2)

[error] string `https://schema.org/Organization` has 3 occurrences, make it a constant

(goconst)